api/src/main/protobuf/nelson/version.proto

```diff
@@ -0,0 +1,27 @@
+//: ----------------------------------------------------------------------------
+//:   Copyright (C) 2018 Verizon.  All Rights Reserved.
+//:
+//:   Licensed under the Apache License, Version 2.0 (the "License");
+//:   you may not use this file except in compliance with the License.
+//:   You may obtain a copy of the License at
+//:
+//:       http://www.apache.org/licenses/LICENSE-2.0
+//:
+//:   Unless required by applicable law or agreed to in writing, software
+//:   distributed under the License is distributed on an "AS IS" BASIS,
+//:   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//:   See the License for the specific language governing permissions and
+//:   limitations under the License.
+//:
+//: ----------------------------------------------------------------------------
+syntax = "proto3";
+
+package nelson.api;
+
+option go_package = "nelson";
+
+message Version {
+  int32 series = 1;
```

Comment:
I always wondered why we don't call this major, minor, and patch, per semantic versioning https://semver.org/

Reply:
ha, we actually did this for a reason. This way its more obvious what the fields mean - we would continually run into people who were like "my understanding of `minor` is X" so we were like "this is the feature version"